src/lib/radio/radio.ts

```diff
@@ -364,9 +401,10 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
   get disabled(): boolean {
     return this._disabled || (this.radioGroup != null && this.radioGroup.disabled);
   }
-
   set disabled(value: boolean) {
     this._disabled = coerceBooleanProperty(value);
+    // Update rippleDisabled
+    this._changeDetector.markForCheck();
```

Comment:
Could you elaborate on what this comment means? I don't see why you would need this in the setter since `disabled` is an `@Input`